api/src/main/java/org/infinispan/api/mutiny/MutinyStreamingCache.java

```diff
@@ -35,16 +36,43 @@
     * @param key      key to use
     * @param backPressureStrategy the {@link BackPressureStrategy} to apply
     */
-   MultiEmitter<ByteBuffer> put(K key, BackPressureStrategy backPressureStrategy);
+   default MultiEmitter<ByteBuffer> put(K key, BackPressureStrategy backPressureStrategy) {
+      return put(key, CacheEntryExpiration.IMMORTAL, backPressureStrategy);
+   }
+
+   /**
+    * Initiates a streaming put operation. It is up to the application to write to the returned {@link OutputStream}
```

Comment:
`It is up to the application to write to the returned {@link OutputStream}` there is no `OutputStream`

Reply:
Fixed